src/depth.json

```diff
@@ -29,7 +32,7 @@
       "deprecated": false
     },
     "modal": {
-      "value": "{depth.300.value}",
+      "value": "{depth.400.value}",
```

Comment:
I'd argue that `dialog` and `modal` are one in the same. So, please update both to be the same level and mark `modal` as deprecated. 

The primary token for use will be `dialog`. 

Dialog is a thing. Modal is a state.

Reply:
@BrentLabasan have you had an opportunity to address this?